sycl/test/aie/basic_buffer.cpp

```diff
@@ -0,0 +1,27 @@
+// REQUIRES: aie
+
+// RUN: %aie_clang %s -o %t.bin
+// RUN: %if_run_on_device %run_on_device %t.bin > %t.check 2>&1
+// RUN: %if_run_on_device FileCheck %s --input-file=%t.check
+
+#include "aie.hpp"
+
+int main() {
+  constexpr std::size_t size = 10;
+  aie::device<3, 3> dev;
+  aie::buffer<int> buff;
+  for (int i = 0; i < size; i++)
+    buff.push_back(i);
```

Comment:
Sure but it does not prevent a SYCL-like
```suggestion
  aie::buffer<int> buff { size };
  std::iota(buff.begin(), buff.end(), 0);
```
or even better the range iota. :-)